index.d.ts

```diff
@@ -27,5 +27,6 @@ declare module 'datadog-iast-taint-tracking' {
         concat(transactionId: string, result: string, op1: string, op2: string): string;
         trim(transactionId: string, result: string, thisArg: string): string;
         trimEnd(transactionId: string, result: string, thisArg: string): string;
+        substring(transactionId: string, subject: string, result: string, start: number, end: number): string;
```

Comment:
here also, I should change the order of the params:
```suggestion
        substring(transactionId: string, result: string, subject: string, start: number, end: number): string;
```